README.md

```diff
@@ -8,10 +8,14 @@
 Trivial library for correct and stable conversions between `Task` and `Future`.  To use:
 
 ```sbt
-libraryDependencies += "io.verizon.delorean" %% "core" % "1.1.37"
+// For Scalaz 7.1.x
+libraryDependencies += "io.verizon.delorean" %% "core" % "1.1.37-scalaz-7.1"
```

Comment:
Oh you cant do it like this. As you have it here, both will be on the classpath at the same time, which will cause a runtime binary collision. You need to dynamically include one or the other, based on the matrix params.